backend/src/apiserver/main.go

```diff
@@ -235,10 +244,25 @@ func startHttpProxy(resourceManager *resource.ResourceManager) {
 	// Register a handler for Prometheus to poll.
 	topMux.Handle("/metrics", promhttp.Handler())
 
+	// Register webhook
+	registerWebhook(topMux, controllerClient)
+
 	http.ListenAndServe(*httpPortFlag, topMux)
 	glog.Info("Http Proxy started")
 }
 
+func startWebhookHTTPProxy(controllerClient ctrlclient.Client) {
+	glog.Info("Starting Http Proxy in Kubernetes Webhook Mode")
+
+	topMux := mux.NewRouter()
+
+	// Register the webhook
+	registerWebhook(topMux, controllerClient)
+
+	http.ListenAndServe(*httpPortFlag, topMux)
+	glog.Info("Http Proxy started in Kubernetes Webhook Mode")
```

Comment:
I think the previous function call is blocking until HTTP server stops, so I think this log message might be stale once it gets logged.

Reply:
Good catch! I will remove it.